examples/v16/central_system.py

```diff
@@ -31,10 +31,22 @@ def on_boot_notitication(self, charge_point_vendor: str, charge_point_model: str
 
 
 async def on_connect(websocket, path):
-    """ For every new charge point that connects, create a ChargePoint instance
-    and start listening for messages.
-
+    """ For every new charge point that connects, create a ChargePoint
+    instance and start listening for messages.
     """
+    requested_protocols = websocket.request_headers['Sec-WebSocket-Protocol']
+    if bool(websocket.subprotocol):
+        logging.info("Protocols Matched: %s", websocket.subprotocol)
```

Comment:
Wouldn't `if websocket.subprotocol == '2.0.1` be more explicit?

Reply:
For this example in specific, where I had used just one supported lib, yes. But in general, you can support several, so if the result of the subprotocol is positive, then you can continue with the communication, regardless of which subprotocol of the list you have chosen

Reply:
Ok. Fair point.